### ISSUE TITLE
A bit more about: resampling_fn_probabilities

### ISSUE THREAD

**Issue by imruljubair** [MIC-DKFZ/nnUNet#1931]
Hi,

Can anyone tell me the significance of `resampling_fn_probabilities` with an example. I want to learn its use cases to check whether it suits my dataset or not.

Looking forward.

**Comment by sten2lu** [comment]
Hi @imruljubair,

`resampling_fn_probabilities` is used to resize logits or class probabilities to the desired shape. 
A typical usecase of this function can be seen in the [inference code](https://github.com/MIC-DKFZ/nnUNet/blob/master/nnunetv2/inference/export_prediction.py#L15). 

Best regards,
Carsten
